### PR TITLE
Fix typos schema references

### PR DESCRIPTION
**Pull request by apinzenoehler**
Unification of schema references concerning grouping and spelling.

Low priority documentation issues.

### CODE REVIEW

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcMeasureResource/Documentation.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ The _IfcMeasureResource_ schema specifies units and defined measure types that m
 In different types of units there are five basic cases:
 
 * Basic SI-units, which cover a number of fundamental units of mainly physical quantities defined by [ISO-1000](../../bibliography.htm#iso-1000){ .int-ref} such as meter or millimeter as unit for length measure or square meter as a unit for area measure. The unit may have a scaling prefix (for example: milli, kilo).
-* Derived SI-units, which are defined as a derivation of the basic SI-units, for example, Newton (kg m / s2) as a unit of force. Both basic and derived SI-units are in the IFCs represented by IfcSIUnit.
+* Derived SI-units, which are defined as a derivation of the basic SI-units, for example, Newton (kg m / s2) as a unit of force. Both basic and derived SI-units are in the IFCs represented by IfcSIUnit
```
Comment (pjanck) [inline IFC4x3/Sections/Resource definition data schemas/Schemas/IfcMeasureResource/Documentation.md:10]:
```suggestion
* Derived SI-units, which are defined as a derivation of the basic SI-units, for example, Newton (kg m / s2) as a unit of force. Both basic and derived SI-units are in the IFCs represented by IfcSIUnit.
```

I don't understand the reason for this change. All other bullet points end with a `.`.

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcMeasureResource/Functions/IfcDimensionsForSiUnit/Expression.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(n : IfcSiUnitName )     : IfcDimensionalExponents;
+(n : IfcSIUnitName )     : IfcDimensionalExponents;
```
Comment (pjanck) [inline IFC4x3/Sections/Resource definition data schemas/Schemas/IfcMeasureResource/Functions/IfcDimensionsForSiUnit/Expression.txt:1]: I'd change the file name as well.

**Review by pjanck — APPROVED** [review]
Two suggestions. Otherwise LFTM.

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedComponentElements/DocSchema.xml [hunk 3]
```diff
@@ -12,7 +12,6 @@
 				<DocDefinitionRef Name="IfcElementType" UniqueId="da39bfeb-9b18-421f-aeef-45f9c7dff83d" />
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedComponentElements/DocSchema.xml:12]: Indentation seems wrong. Did you use spaces and tabs?

**Review by SergejMuhic — APPROVED** [review]
Looks fine to me. Just not sure what happened to the indentation in `IfcSharedComponentElements`.

**Pull request merged**